CMakeLists.txt

```diff
@@ -511,13 +511,24 @@ endif()
 #   - place the CMakeLists.txt in the first-level subdirectory, e.g.
 #     Externals/zlib/CMakeLists.txt (that is: NOT in some Src/ subdirectory)
 #
+include_directories("${CMAKE_CURRENT_BINARY_DIR}/Externals")
 if (_M_X86)
   add_subdirectory(Externals/Bochs_disasm)
 endif()
 add_subdirectory(Externals/cpp-optparse)
 add_subdirectory(Externals/glslang)
 add_subdirectory(Externals/imgui)
 
+set(DISABLE_FRONTENDS ON CACHE BOOL "" FORCE)
+set(DISABLE_DEPS ON CACHE BOOL "" FORCE)
+set(USE_SQLITE3 OFF CACHE BOOL "" FORCE)
+set(USE_PNG OFF CACHE BOOL "" FORCE)
+set(USE_ZLIB OFF CACHE BOOL "" FORCE)
+set(BUILD_STATIC ON CACHE BOOL "" FORCE)
+set(BUILD_SHARED OFF CACHE BOOL "" FORCE)
```

Comment:
And now, you've changed the option for all the libraries.
You should move this to another file so that it doesn't leak to this project or make the main CMakeLists.txt even longer.